sdks/java/io/kafka/pom.xml

```diff
@@ -40,6 +40,35 @@
             <skip>true</skip>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-shade-plugin</artifactId>
+          <executions>
+            <execution>
+              <phase>package</phase>
+              <goals>
+                <goal>shade</goal>
+              </goals>
+              <configuration>
+                <artifactSet>
+                  <includes>
+                    <include>com.google.guava:guava</include>
+                  </includes>
+                </artifactSet>
+                <relocations>
```

Comment:
@davorbonaci 

IIUC you mean you want to make sure that every class that was shaded into the jar (And every class packaged into the jar) begins with `org.apache.beam.`?